pkg/crane/options.go

```diff
@@ -54,3 +54,11 @@ func WithTransport(t http.RoundTripper) Option {
 func Insecure(o *options) {
 	o.name = append(o.name, name.Insecure)
 }
+
+// WithAuth is a functional option for overriding the default authentication
+// mechanism for GCR operations.
```

Comment:
small nit: can you change `GCR` to just `remote`? This isn't GCR-specific.

Reply:
Updated from `GCR` to `remote`